server/src/service/infrastructure/networkService.js

```
@@ -470,6 +472,57 @@ class NetworkService {
 		}
 	}
 
+	/**
+	 * Requests the status of a game monitor.
+	 *
+	 * @param {Object} monitor - The monitor object to request the status for.
+	 * @returns {Promise<Object>} The response from the game status request.
+	 * @throws {Error} Throws an error if the request fails or if the monitor is not configured correctly.
+	 * @property {string} monitorId - The ID of the monitor.
+	 * @property {string} type - The type of the monitor (should be "game").
+	 * @property {number} responseTime - The time taken for the request.
+	 * @property {Object|null} payload - The game state response or null if the request failed.
+	 * @property {boolean} status - Indicates if the request was successful (true) or not (false).
+	 * @property {number} code - The status code of the request (200 for success, NETWORK_ERROR for failure).
+	 * @property {string} message - A message indicating the result of the request.
+	 */
+	async requestGame(monitor) {
+		try {
+			const { url, port, gameId } = monitor;
+
+			const gameResponse = {
+				code: 200,
+				status: true,
+				message: "Success",
+				monitorId: monitor._id,
+				type: "game",
+			};
+
+			const state = await GameDig.query({
+				type: gameId,
+				host: url,
+				port: port,
+			}).catch((error) => {
+				null;
```

We should avoid swallowing errors, let's at least log this so we know something went wrong